assets/dev/js/editor/utils/controls-css-parser-helper.js

```diff
@@ -0,0 +1,31 @@
+export default class ControlsCSSParserHelper {
```

Comment:
@udidol You are correct. This has been separated to make the code testable. I also spoke to Nevo about this, here was his comment:

```
Hi, first it will be better to not use jQuery in new code
It will be easer for testing and easier for removing jQuery later
maybe just extract that function to another file, and import it on the old file
and then check only yours
```

I am open to further discussion 😃